plugins/modules/aci_outofsvc_interface_blacklist.py

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'certified'}
+
+DOCUMENTATION = r'''
+---
+module: aci_outofsvc_interface_blacklist
+short_description: Enabling or Disabling physical interfaces.
+description:
+- Manages enabling and disabling physical interfaces on Cisco ACI fabrics.
+options:
+  pod_id:
+    description:
+    - ID of the pod eq. 1
+    type: int
+    required: true
```

Comment:
Not making pod_id, node_id and interface required would allow the module to be able to query all disabled interface.